src/FSharp.Control.TaskSeq/TaskSeq.fsi

```diff
@@ -154,414 +175,872 @@ module TaskSeq =
     /// <exception cref="T:ArgumentNullException">Thrown when either of the input sequences is null.</exception>
     val prependSeq: source1: seq<'T> -> source2: taskSeq<'T> -> taskSeq<'T>
 
-    /// Returns taskSeq as an array. This function is blocking until the sequence is exhausted and will properly dispose of the resources.
+    /// <summary>
+    /// Builds an F# <see cref="list" /> from the input task sequence in <paramref name="source" />.
+    /// This function is blocking until the sequence is exhausted and will then properly dispose of the resources.
+    /// </summary>
+    ///
+    /// <param name="source">The input task sequence.</param>
+    /// <returns>The resulting list.</returns>
+    /// <exception cref="T:ArgumentNullException">Thrown when the input sequence is null.</exception>
     val toList: source: taskSeq<'T> -> 'T list
 
-    /// Returns taskSeq as an array. This function is blocking until the sequence is exhausted and will properly dispose of the resources.
+    /// <summary>
+    /// Builds an <see cref="array" /> from the input task sequence in <paramref name="source" />.
+    /// This function is blocking until the sequence is exhausted and will then properly dispose of the resources.
+    /// </summary>
+    ///
+    /// <param name="source">The input task sequence.</param>
+    /// <returns>The resulting array.</returns>
+    /// <exception cref="T:ArgumentNullException">Thrown when the input sequence is null.</exception>
     val toArray: source: taskSeq<'T> -> 'T[]
 
     /// <summary>
-    /// Returns the task sequence <paramref name="source" /> as an F# <see cref="seq" />, that is, an
-    /// <see cref="IEnumerable&lt;'T>" />. This function is blocking at each <see cref="yield" />, but otherwise
-    /// acts as a normal delay-executed sequence.
-    /// It will then dispose of the resources.
+    /// Views the task sequence in <paramref name="source" /> as an F# <see cref="seq" />, that is, an
```

Comment:
This 'Views' might speak to you, but is kinda verbing [(see intro of this)](https://wordsmith.org/words/primary.html) and might be confusing to various readers, i.e. yes its a Lens and the lib in general is lazy, but the reader might be trying to figure out what the operator _actually does_, which is provide/decorate an input sequence, mapping it to a different result with a given mapping semantic.

(Coming back to here after seeing 20 of them - it probably makes sense for this one, but finding a different term might be more explict/clear overall?)

Reply:
I took this from the way `seq` documentation is done. Leaving it in for now. We can do another pass over the documentation at some point, for sure, though.